CHANGELOG.md

```diff
@@ -9,6 +9,29 @@ From v1.2, Besu requires Java 11.  Besu on Java 8 is no longer supported.
 In v1.2, we removed the entry-point script from our Docker image. Refer to the [migration guide](https://besu.hyperledger.org/en/latest/Deploying-Besu/High-Availability/)
 for information on options that were previously automatically added to the Besu command line. 
 
+## 1.2.4 
+
+### Additions and Improvements 
+
+- Add Istanbul block for Rinkeby [\#35](https://github.com/hyperledger/besu/pull/35)
+- Add Istanbul block for Goerli [\#27](https://github.com/hyperledger/besu/pull/27) 
+- Add Istanbul block for Ropsten [\#26](https://github.com/hyperledger/besu/pull/26) 
```

Comment:
Add Istanbul fork block (6485846) for Ropsten.

Reply:
Done